net/parse.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package net
+
+import (
+	forkednet "k8s.io/utils/third_party/forked/golang/net"
```

Comment:
given these are locked at a point in time, should we name the package `.../forked/golang_116/net` or something?

Reply:
I also wonder if this is a good place to use `k8s.io/utils/internal/...` so things that import this repo don't use our third_party package directly

Reply:
I loooove the idea of internal.  Will do that!  We should do that with the forked reflect too

Reply:
As for naming - it's not quite 1.16 - it's more likely that it will be 1.17 (as seldom as this code is touched).  But if it WERE changed, we could update it, so it's not 16 or 17 any more.

Reply:
yeah, if we're not exporting it I care way less about the exact package name